# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: content/events/src/nsEventStateManager.cpp
# Commit: b3f0e5c7874d
# Full Hash: b3f0e5c7874dbd470d96e476e116740e4ca35925
# Author: David Zbarsky <dzbarsky@gmail.com>
# Date: 2012-08-06 14:40:43
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684: Create a stack-based nsFrameIterator class out of nsIFrameTraversal and nsIFrameEnumerator r=roc
# ==============================================================================

diff -r 4bc7154190dd -r b3f0e5c7874d content/events/src/nsEventStateManager.cpp
--- a/content/events/src/nsEventStateManager.cpp	Sun Aug 05 22:55:18 2012 -0400
+++ b/content/events/src/nsEventStateManager.cpp	Sun Aug 05 23:00:55 2012 -0400
@@ -65,7 +65,6 @@
 #include "nsCaret.h"
 
 #include "nsSubDocumentFrame.h"
-#include "nsIFrameTraversal.h"
 #include "nsLayoutCID.h"
 #include "nsLayoutUtils.h"
 #include "nsIInterfaceRequestorUtils.h"
@@ -112,8 +111,6 @@
 
 #define NS_USER_INTERACTION_INTERVAL 5000 // ms
 
-static NS_DEFINE_CID(kFrameTraversalCID, NS_FRAMETRAVERSAL_CID);
-
 static bool sLeftClickOnly = true;
 static bool sKeyCausesActivation = true;
 static PRUint32 sESMInstanceCount = 0;